### PR TITLE
Remove downloaded tarballs from the prowbazel image.

### PR DESCRIPTION
**Pull request by PiotrSikora**
Signed-off-by: Piotr Sikora <piotrsikora@google.com>

### CODE REVIEW

**Review by clarketm — APPROVED** [review]

**Comment by PiotrSikora** [comment]
Is tide broken?

**Comment by clarketm** [comment]
@PiotrSikora 
> Is tide broken?

I think tide is awaiting an approval from someone in @istio/wg-test-and-release-maintainers

**Review by howardjohn — APPROVED** [review]

**Pull request merged**
